code/modules/mob/living/blood.dm

```diff
@@ -141,29 +152,29 @@
 	if(iscarbon(AM))
 		var/mob/living/carbon/C = AM
 		if(blood_id == C.get_blood_id())//both mobs have the same blood substance
-			if(blood_id == "blood") //normal blood
+			if(blood_id == "blood" || blood_id == "jellyblood") //normal blood
 				if(blood_data["viruses"])
 					for(var/thing in blood_data["viruses"])
 						var/datum/disease/D = thing
 						if((D.spread_flags & DISEASE_SPREAD_SPECIAL) || (D.spread_flags & DISEASE_SPREAD_NON_CONTAGIOUS))
 							continue
 						C.ForceContractDisease(D)
 				if(!(blood_data["blood_type"] in get_safe_blood(C.dna.blood_type)))
-					C.reagents.add_reagent("toxin", amount * 0.5)
-					return 1
+					C.reagents.add_reagent("bonehurtingjuice", amount * 0.5)
```

Comment:
What's the reason for this change? Wouldn't it be better to add a different variant of toxin that deals toxinloss that ignores the toxinlover flag instead of changing this to a meme chem that makes people spam "oof", "ouch", and "owie"?

Reply:
Slimepeople could end up using any blood for free, massive heals. this was a better solution that still held a viable penalty for not paying attention to your blood type.

Reply:
is there anything that's stopping you from just... using adjusttoxloss straight up without a reagent inbetween